### PR TITLE
Optimize AssetService

### PR DESCRIPTION
**Pull request by chimp1984**
Previous version had several performance issues:
- We kept tradeStatistics in a lookup map in memory (about 40 MB)
- We built up the StatefulAssets even if it was not used/needed.
Building the StatefulAssets was an expensive call.
- AssetTradeActivityCheck was only used for printing out statistics and
called on startup causing performance cost.

Optimized version:
- We call build the StatefulAssets only on demand (user activates
view). Method call takes now 22 ms.
- We create a optimized lookup map with a tuple of the data we need
and keep it as local variable.
- We removed AssetTradeActivityCheck.

Thanks to the profiling reporting from:
https://github.com/bisq-network/proposals/issues/199

### CODE REVIEW

File: core/src/main/java/bisq/core/dao/governance/asset/AssetService.java
Comment (sqrrm) [inline core/src/main/java/bisq/core/dao/governance/asset/AssetService.java:240]:
```suggestion

    // Get the trade volume from lookupDate until current date
```

File: desktop/src/main/java/bisq/desktop/main/dao/burnbsq/assetfee/AssetFeeView.java
Comment (sqrrm) [inline desktop/src/main/java/bisq/desktop/main/dao/burnbsq/assetfee/AssetFeeView.java]:
```suggestion
```

**Review by sqrrm — CHANGES_REQUESTED** [review]
Looks good. Would prefer to have pure refactoring as separate commits for easier review.

Added a couple of suggestions before merging.

Needs review from @ManfredKarrer since it touches core/dao

**Comment by chimp1984** [comment]
> Would prefer to have pure refactoring as separate commits for easier review.

Agree. Sorry.

**Comment by ManfredKarrer** [comment]
utACK

**Review by sqrrm — APPROVED** [review]
ACK

**Pull request merged**
